node-check/osgvo-additional-htcondor-config

```diff
@@ -68,7 +68,7 @@ chmod 755 $STASHCP
 if ($STASHCP /osgconnect/public/dweitzel/stashcp/test.file stashcp-test.file) &>/dev/null; then
 
     cat >stashcp_plugin <<EOF
```

Comment:
While you are at it replacing this whole here-doc, i feel like suggesting

```suggestion
    cat >stashcp_plugin <<"EOF"
```

and then removing all the makes-it-harder-to-read backslash escaping, like it's going out of style.

Reply:
OMG, that's amazing.  Will do.

Reply:
Actually... can't do that (I think?) because I should actually be referring to a variable from the prior line...

Reply:
Ah, bummer.  I didn't notice the unescaped `$STASHCP` in there.  Yeah i guess you need to quote everything else if you want variable expansion.

Reply:
Ah - that's because it wasn't there until your review made me go look for it.  I had forgotten to copy it over.

So, you know, the process works!

Reply:
Yeah, you don't even need to assign me as a reviewer!

Alls you need is to swap out bash for sh, and the spider sense goes off ;)